independent-publisher-connectors/Xero Projects/apiDefinition.swagger.json

```diff
@@ -0,0 +1,582 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",

```

Comment:
@troystaylor Can you please provide your connector name as title.

Suggestion : `Xero Projects`